src/execution_plan/execution_plan_build/build_match_op_tree.c

```diff
@@ -84,7 +84,7 @@ static void _ExecutionPlan_ProcessQueryGraph
 
 			// resolve source node by performing label scan
 			QGNode *clone = QGNode_Clone(src);
-			NodeScanCtx ctx = NODE_CTX_NEW(alias, label, label_id, clone);
+			NodeScanCtx *ctx = NodeScanCtx_New((char *)alias, (char *)label, label_id, clone);
```

Comment:
Didn't we wanted `NodeScanCtx_New` to perform the clone internally ?
`NodeScanCtx *ctx = NodeScanCtx_New((char *)alias, (char *)label, label_id, src);`